Cos3rd.java

```diff
@@ -1,13 +1,30 @@
 class Cos3rd extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * Finding the third side of a triangle using two sides and their contained angle
+  * @author: Alina
   */
   
   public void run() {
     
-    // start coding here
+    // Variables
+    double dblSideA;
+    double dblSideB;
+    double dblAngleC;
+
+    double dblSideC;
```

Comment:
I put a space between these variables because the later one is used in a separate section.